pages/price-feeds/use-real-time-data/solana.mdx

```diff
@@ -115,16 +127,16 @@ const solUsdPriceFeedAccount = pythSolanaReceiver
   .toBase58();
 ```
 
-The price feed account integration assumes that an off-chain process is continuously updating each price feed.
-The Pyth Data Association sponsors price updates for a subset of commonly-used price feeds on shard 0.
+**NOTE:** The Price Feed Accounts integration assumes that an off-chain process is continuously updating each price feed.
```

Comment:
Notes seem inconsistent. 
Sometimes it's written as NOTE sometime as Note.
Sometimes it's inside a Callout and sometimes not.